scripts/elasticsearch-old.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Could you rename this script to make it a bit more obvious what it does without having to read it? eg. `install-elasticsearch-old.sh`. Same with the other script.

Reply:
It both installs and runs Elasticsearch (daemonized). I figured just `elasticsearch.sh` was a fair name for it. I could rename it to `install-and-run-old-elasticsearch.sh`...

Reply:
Heh, I suppose that would be a bit of a mouthful. Hmm, let's leave it as-is then.